gui/controls/FpsItem.qml

```diff
@@ -34,7 +37,7 @@ Rectangle {
         Image {
             id: spinnerImage
             anchors.verticalCenter: parent.verticalCenter
-            source: "images/Logo.png"
+            source: "qrc:/images/Logo.png"
```

Comment:
I noticed this rotating thing - can we remove it, or only have it in some debug mode? When opening the ventilator UI, it looks kinda confusing.

Reply:
This is supposed to show on debugging more only.  The reason for the image is to force the animation to trigger, which increments the FPS count. Maybe I can try with opacity 0 to see if it is less distracting. Another option is to collect FPS on the C++ side, but it is more work.